### ISSUE TITLE
Has Model.net been trained on CityScape extra training Data ?

### ISSUE THREAD

**Issue by ghost** [e-lab/ENet-training#12]
Just wondering whether or not segmentation accuracy would be better with including the CityScapes extra set to the training data.

**Comment by codeAC29** [comment]
No, we did not use extra training set. Ideally the more data you use, the better your network performance will be.
